modules/wizard/wizard.php

```diff
@@ -704,10 +704,10 @@ public function save_step_untranslated_contents() {
 
 		while ( $nolang = $this->model->get_objects_with_no_lang( 1000 ) ) {
 			if ( ! empty( $nolang['posts'] ) ) {
-				$this->model->set_language_in_mass( 'post', $nolang['posts'], $language->slug );
+				$this->model->post->set_language_in_mass( 'post', $nolang['posts'], $language->slug );
 			}
 			if ( ! empty( $nolang['terms'] ) ) {
-				$this->model->set_language_in_mass( 'term', $nolang['terms'], $language->slug );
+				$this->model->term->set_language_in_mass( 'term', $nolang['terms'], $language->slug );
 			}
```

Comment:
Here and in `settings.php` we'll need to "set language in mass" for all types, not only for `post` and `term`.
I suggest do do that in another PR, maybe in the one dealing with `get_objects_with_no_lang()`.

Reply:
OK to solve this in another PR.

Reply:
I completed #1163 issue.